SpringProject/src/it/java/com/gradproject2019/user/UserControllerIT.java

```diff
@@ -141,6 +144,28 @@ public void shouldReturn401WhenUnauthorisedEdit() throws URISyntaxException {
         Assert.assertEquals("User unauthorized to perform action.", response.getBody().getMessage());
     }
 
+    @Test
+    public void shouldReturn200AndEmptyListWhenDatabaseEmpty() throws URISyntaxException {
+        URI uri = new URI(baseUri);
+        clearRepositories();
```

Comment:
Can't see it in this PR, but this is happening in the 'before' step for every test so this is unnecessary?

Reply:
This is for a test where we don't want anything in the database, so anything added to the database in the universalSetup() should be cleared before this test